cloudbuild/cloudbuild.yaml

```diff
@@ -1,10 +1,10 @@
 steps:
-# 1. Create a Docker image containing hadoop-connectors repo
+# 0. Create a Docker image containing hadoop-connectors repo
```

Comment:
I started the numbering a #0 to match the way that Cloud Build numbers the steps